packages/frontend/pages/index.tsx

```diff
@@ -1,23 +1,26 @@
-import { connect } from "react-redux";
-import { Dispatch } from 'redux';
+import { ReactElement } from 'react';
+import { connect } from 'react-redux';
 import Head from 'next/head';
 import Link from 'next/link';
 import { Button, Form } from 'semantic-ui-react';
-import { GOLD, GOLD_HOVER, ColourTheme, DAY_COLOURS } from '../components/colours';
 
 import Nav from '../components/nav';
-import { changeTheme, CustomAction } from '../redux/actions';
-import { State, ReducersState } from '../redux/reducers';
+import { RootState } from '../store/index';
+import { ThemeOptions, UserOptionsState } from '../store/userOptions/types';
+import { setTheme } from '../store/userOptions/actions';
+import { COMMON_COLORS } from '../components/colors';
+
+const { GOLD, GOLD_HOVER } = COMMON_COLORS;
 
 interface Props {
-  theme?: ColourTheme;
-  handleThemeChange?: () => void;
+  theme: ThemeOptions;
+  setTheme: typeof setTheme;
 }
 
-const Home = (props: Props): React.ReactElement => {
-  const { handleThemeChange } = props;
-  let { theme } = props;
-  theme = theme || DAY_COLOURS;
+const Home = (props: Props): ReactElement => {
+  const { theme } = props;
+  // eslint-disable-next-line no-shadow
```

Comment:
Oops, I forgot to remove this. Can you add this to the eslint config rules to ignore? Thanks.

Reply:
Actually see if you can make the code nicer so that we can still have this rule in place. Might be good to have the no-shadow rule.